scimath/Mathematics/FitToHalfStatistics.tcc

```diff
@@ -128,6 +128,7 @@ void FitToHalfStatistics<CASA_STATP>::_getRealMinMax(
         CountedPtr<AccumType>& realMin, CountedPtr<AccumType>& realMax,
     CountedPtr<AccumType> knownMin, CountedPtr<AccumType> knownMax
 ) {
+    _setRange();
```

Comment:
if its needed here, shouldn't it also be needed in `getMinMax` and `getQuantiles`?